### PR TITLE
Fix a typo in SplitQueryFn error handling

### PR DESCRIPTION
**Pull request by vikkyrk**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---
- Also add unit test to catch this bug.

### CODE REVIEW

**Comment by vikkyrk** [comment]
R: @aaltay

File: sdks/python/apache_beam/io/datastore/v1/datastoreio_test.py [hunk 1]
```diff
@@ -122,6 +122,36 @@ def test_SplitQueryFn_with_query_limit(self):
...
+        with patch.object(query_splitter, 'get_splits',
+                          side_effect=ValueError("Testing query split error")):
+          split_query_fn = ReadFromDatastore.SplitQueryFn(
+              self._PROJECT, self._query, None, num_splits)
+          mock_context = MagicMock()
+          mock_context.element = self._query
+          split_query_fn.start_bundle(mock_context)
+          returned_split_queries = []
+          for split_query in split_query_fn.process(mock_context):
+            print split_query
```
Comment (aaltay) [inline sdks/python/apache_beam/io/datastore/v1/datastoreio_test.py]: Use `logging` instead of `print`

**Comment by aaltay** [comment]
LGTM

File: sdks/python/apache_beam/io/datastore/v1/datastoreio_test.py (hunk 1 above)
Comment (vikkyrk) [inline sdks/python/apache_beam/io/datastore/v1/datastoreio_test.py]: Fixed

**Comment by vikkyrk** [comment]
R: @robertwb

**Pull request closed without merging**